PiwikTracker/Dispatcher.swift

```diff
@@ -4,7 +4,9 @@ public protocol Dispatcher {
     
     var baseURL: URL { get }
     
-    var userAgent: String? { get }
+    var userAgent: String? { get set }
+    
+    func setUserAgent(ua: String)
```

Comment:
It is not necessary to define a specific setter function here, right? I think it is sufficient to use the setter of the property.